packages/react/src/components/CardCodeEditor/__snapshots__/CardCodeEditor.story.storyshot

```diff
@@ -7,7 +7,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/C
   <div
     className="bx--modal is-visible iot--editor iot--composed-modal--large iot--composed-modal"
     data-floating-menu-container={true}
-    data-testid="ComposedModal"
```

Comment:
@davidicus This is another item I questioned. Technically, the `testId` was being passed down before, so the CardCodeEditor ended up with the default `ComposedModal` data-testid nested deep down...Should I _not_ pass it down and save that for v3, so that items like this aren't changed?

Reply:
done.